### DISCUSSION TITLE
PAG 初始化失败，如需技术支持可到 https://github.com/Tencent/libpag/discussions 提问 请问这是啥问题

### DISCUSSION THREAD

**Discussion by wardenlzr** [Tencent/libpag#1715] [General] [answered]
![image](https://github.com/Tencent/libpag/assets/16572346/19217c1b-7aab-41b9-8c33-607e204584a0)

**Comment by domchen** [comment, marked as answer]
这个也页面引用的libpag的库是托管在免费的CDN上的，你所在的网络访问免费的CDN可能临时挂掉了。
